agent/bench-scripts/pbench-fio

```diff
@@ -871,6 +870,8 @@ function fio_run_benchmark() {
 								fi
 								popd >/dev/null
 							fi
+							debug_log "post-processing tool data"
+							pbench-postprocess-tools --group=$tool_group --dir=$benchmark_results_dir
```

Comment:
This is a bug fix where we are now post-processing tool data when "post-process only" is requested.